swap/prices.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 The Swarm Authors
+// This file is part of the Swarm library.
+//
+// The Swarm library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The Swarm library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the Swarm library. If not, see <http://www.gnu.org/licenses/>.
+
+package swap
+
+/*
+This module contains the pricing for message types as constants.
+
+Pricing in Swarm is defined in an internal unit.
+The name of this internal unit is honey.
+The honey unit allows to set prices of messages relative to each other
```

Comment:
Consider:

Pricing in Swarm is defined as an internal unit (called `honey`). Honey acts as a unit of relative message pricing (that is, it allows to set prices relative to message types, irrespective of a reference currency)

Reply:
Thanks! Updated.